wordpress-mobile/WordPress-iOS
# Login Epilogue: updated button panel view

Ref #13413 

This updates the button panel display (the view behind the Done button). 
- The button panel is the full width of the screen on all devices.
- Removed the shadow that appeared above the button panel when the table was scrollable.
- When the table is scrollable, the button panel is grey/black with a line at the top.
- When the table is not scrollable, the button panel color matches the table, and there is no line.

Not yet updated:
- iPad specific layout, including the Done button width matching the table width.
- Accessibility.

To test:
- Log into the app.
- Verify the button panel behavior is as described above (scrollable vs not).
- Verify the button panel looks like the Zeplin design, or that on #13413 .

|  ![short_list_light](https://user-images.githubusercontent.com/1816888/75485033-3130b280-5967-11ea-8f25-7c118353ff32.png) |  ![short_list_dark](https://user-images.githubusercontent.com/1816888/75485044-3857c080-5967-11ea-8448-7892329c1e6d.png) |
|--------|-------|
|  ![long_list_light](https://user-images.githubusercontent.com/1816888/75485069-43125580-5967-11ea-8b69-2796cee6b508.png) |  ![long_list_dark](https://user-images.githubusercontent.com/1816888/75485079-49a0cd00-5967-11ea-8a15-80ccad9773c7.png) |


- On an iPad, verify the button panel is the full width of the view (it's the same on iPhone, but the difference is only visible on iPad).

| ![ipad_portrait](https://user-images.githubusercontent.com/1816888/75485297-ae5c2780-5967-11ea-83ac-f7d620a1b9b0.png) | ![ipad_landscape](https://user-images.githubusercontent.com/1816888/75485312-b3b97200-5967-11ea-8c15-5f88d294bce9.png)  |
|--------|-------|


PR submission checklist:
